src/App.jsx

```diff
@@ -13,11 +13,11 @@ import ProfilePage from "./pages/settingsPage/profilePage";
 import ForgetPassword from "./pages/ForgetPassword/index";
 import Policy from "./pages/policy";
 import Error404 from "./pages/error404";
-import HowItWorks from "./pages/howItWorks/HowItWorks";
+// import HowItWorks from "./pages/howItWorks/HowItWorks";
```

Comment:
- [ ] KIndly ensure you dont leave any commented code to the code base so that the codebase can be clean, organisedd without causing any form of bugs or merge conflict 👍🏽

Reply:
> * [ ]  KIndly ensure you dont leave any commented code to the code base so that the codebase can be clean, organisedd without causing any form of bugs or merge conflict 👍🏽

Okay